dotnet/src/SemanticKernel.Abstractions/Functions/KernelFunction.cs

```diff
@@ -159,7 +160,7 @@ internal KernelFunction(string name, string description, IReadOnlyList<KernelPar
                 throw new OperationCanceledException($"A {nameof(Kernel)}.{nameof(Kernel.FunctionInvoked)} event handler requested cancellation after function invocation.");
             }
 
-            logger.LogFunctionInvokedSuccess(functionResult.Value);
+            logger.LogFunctionResult(functionResult.Value);
```

Comment:
This just logs the `FunctionResult.Value` so the function name isn't accurate. 

Should we change the function name or really log the `FunctionResult`